index.html

```diff
@@ -277,8 +277,7 @@ <h6>Project</h6>
           <li><a href="pages/devel-version.html">Development version</a></li>
           <li><a href="pages/patches.html">Patches</a></li>
           <li><a href="pages/source-code.html">Source code</a></li>
-          <li><a href="pages/contact.html#ml">Mailing lists</a></li>
-          <li><a href="pages/contact.html#gitter">Chat</a></li>
```

Comment:
why are you removing gitter?

Reply:
From the footer, yes. Gitter is just a subitem of the contact page, and now the contact page is mentioned.

I am not adding GitHub discussions, but at the same time Twitter was not here either.

So the idea is to point the users to the contact page, so they can pick there what dey need.

Reply:
ok. it makes sense. 

but, given that you are doing this big refactoring, could you also update the credit year to 2023? I notice it's still 2022

Reply:
Let's do it on a different PR , as the refactor and fixes are related to the removal of mailing lists (except announce), but not the copyright.

Reply:
What credit year? I was thinking about the copyright, but that one is OK.

Reply:
ok